tour/TourManager.js

```diff
@@ -3,14 +3,9 @@ import { getSteps } from "./steps";
 
 const KEY = "hasSeenTour";
 
-async function createDriver() {
-  const { driver } = await import("driver.js");
-  return driver(createOpts());
-}
-
 export async function startTour() {
-  const driver = await createDriver();
-  driver.drive();
+  const tourDriver = driver(opts);
```

Comment:
here you call driver, but in line 21 you await the call. Is driver an async function or not?